style.css

```diff
@@ -330,3 +334,94 @@ a {
   display: flex;
   flex: 1;
 }
+
+.div-contact {
+  background-image: url("images/Frame.png");
+  background-position: bottom right;
+  background-repeat: no-repeat;
+}
```

Comment:
- [ ] Kindly note that according to the project requirements, you are required to use Flexbox to align the elements of this section. Kindly use Flexbox for this section.

![image](https://user-images.githubusercontent.com/86318284/186464800-ce2c4b8c-502a-4124-97aa-b1264520de12.png)